### PR TITLE
Wiki redirect cleanup

### PR DESCRIPTION
**Pull request by Felanbird**
Pull request checklist:

~~- [ ] `CHANGELOG.md` was updated, if applicable~~ N/A

# Description
Noticed the redirect last week, forgot to actually PR it, found another one earlier.
Might as well just link directly to the wiki.

### CODE REVIEW

File: src/widgets/settingspages/ExternalToolsPage.cpp [hunk 1]
```diff
@@ -106,7 +106,7 @@ ExternalToolsPage::ExternalToolsPage()
         const auto description = new QLabel(
             "You can set custom host for uploading images, like "
             "imgur.com or s-ul.eu.<br>Check " +
-            formatRichNamedLink("https://chatterino.com/help/image-uploader",
+            formatRichNamedLink("https://wiki.chatterino.com/Image%20Uploader/",
```
Comment (Felanbird) [inline src/widgets/settingspages/ExternalToolsPage.cpp:109]:
```suggestion
            formatRichNamedLink("https://wiki.chatterino.com/Image%20Uploader",
```
Doing this from web so I don't have to re-open ide, doing it for  https://github.com/Chatterino/wiki/pull/88#issuecomment-867674003

Comment (ALazyMeme) [reply src/widgets/settingspages/ExternalToolsPage.cpp]: so lazy smh

**Review by zneix — APPROVED** [review]

**Review by xHeaveny — APPROVED** [review]

**Review by Mm2PL — APPROVED** [review]

**Comment by fourtf** [comment]
The links are there so they can be changed (indirection) so it doesn't break in the future.

**Comment by Felanbird** [comment]
> The links are there so they can be changed (indirection) so it doesn't break in the future.

Should we be moving existing wiki links to a chatterino.com alternative so this can always be prevented?

**Pull request closed without merging**